ndk/lib/src/androidTest/jni/ndk-test.cpp

```diff
@@ -0,0 +1,50 @@
+#include <jni.h>
+#include <android/log.h>
+#include <sentry.h>
+
+#define TAG "ndk-sample"
+
+extern "C" {
+
+JNIEXPORT void JNICALL Java_io_sentry_ndk_sample_NdkSample_crash(JNIEnv *env, jclass cls) {
+    __android_log_print(ANDROID_LOG_WARN, TAG, "About to crash.");
+    char *ptr = 0;
+    *ptr += 1;
+}
+
+JNIEXPORT void JNICALL Java_io_sentry_ndk_sample_NdkSample_message(JNIEnv *env, jclass cls) {
+    __android_log_print(ANDROID_LOG_WARN, TAG, "Sending message.");
```

Comment:
i know that this is only test code but why `ANDROID_LOG_WARN`? Same for the other occurrences. I think we should default to `INFO` and only log differently when the context requires it (`WARN` for `"About to crash."` is valid,`ERROR` would be the right for `"Transaction failed to send."`).

Reply:
We just copied over the existing functions from `ndk-sample.cpp` without giving it too much thought, but given that there already are other log levels, it makes sense to change them to the appropriate ones for each log statement.